src/app/identity.service.ts

```diff
@@ -512,6 +531,6 @@ export class IdentityService {
 
   // Respond to a received message
   private respond(window: Window, id: string, payload: any): void {
-    window.postMessage({ id, service: "identity", payload }, "*");
+    // window.postMessage({ id, service: "identity", payload }, "*");
```

Comment:
should I remove it?

Reply:
This should be deleted. The library handles this now.